packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -154,4 +155,70 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
     (this[$sourceObject] as GLTFMaterial).emissiveFactor = rgb;
     this[$onUpdate]();
   }
+
+  setAlphaCutoff(cutoff: number): void {
+    cutoff = Math.min(1.0, cutoff);
+    for (const material of this[$correlatedObjects] as
+         Set<MeshStandardMaterial>) {
+      material.alphaTest = cutoff;
+      material.needsUpdate = true;
+    }
+    (this[$sourceObject] as GLTFMaterial).alphaCutoff = cutoff;
+    this[$onUpdate]();
+  }
+
+  getAlphaCutoff(): number {
+    return (this[$sourceObject] as GLTFMaterial).alphaCutoff ??
+        this[$correlatedObjects]!.values().next().value.alphaTest;
+  }
+
+  setDoubleSided(doubleSided: boolean): void {
+    for (const material of this[$correlatedObjects] as
+         Set<MeshStandardMaterial>) {
+      // When double-sided is disabled gltf spec dictates that Back-Face culling
+      // must be disabled, in three.js parlance that would mean FrontSide
+      // rendering only.
+      // https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#double-sided
+      material.side = doubleSided ? DoubleSide : FrontSide;
+      material.needsUpdate = true;
+    }
+    (this[$sourceObject] as GLTFMaterial).doubleSided = doubleSided;
+    this[$onUpdate]();
+  }
+
+  getDoubleSided(): boolean {
+    return (this[$sourceObject] as GLTFMaterial).doubleSided ??
+        this[$correlatedObjects]!.values().next().value.side === DoubleSide;
+  }
+
+  setAlphaMode(alphaMode: AlphaMode): void {
+    for (const material of this[$correlatedObjects] as
+         Set<MeshStandardMaterial>) {
+      if (alphaMode === 'OPAQUE') {
+        material.blending = NoBlending;
```

Comment:
I still don't buy this. Have you looked at how three.js sets up materials based on glTFLoader? They do not seem to change `blending`, but they do change `transparent` and `depthWrite`: https://github.com/mrdoob/three.js/blob/dev/examples/jsm/loaders/GLTFLoader.js#L3092-L3111

Reply:
Hmm, I see. I guess there are multiple ways to set this, changing the blend mode does work in practice, I've been verifying it against the AlphaBlendTest model. But it looks like three wants us to use the transparent setting.

Reply:
Setting depth write to false for transparent objects seems like a hammer,  that's why they are sorted back-to-front, but I guess it's the best way to ensure minimal artifacts on tricky models.

Reply:
Yeah, three is pretty complicated (and has duplicate features that get deprecated over time), but we'd better stick very close to what GLTFLoader does, or we're likely to hit strange edges cases. And yes, there was a long debate about depthWrite false.